internal/common/env.go

```diff
@@ -0,0 +1,9 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package common
+
+const (
+	ElasticPackageEnvPrefix = "ELASTIC_PACKAGE_"
```

Comment:
This way somebody can overwrite the `ElasticPackageEnvPrefix`. I would wrap with a dedicated function, for example: `withElasticPackageEnv(...)`.

Reply:
Updated :+1: 
As a note, I had to change some constants to variables as a consequence of being this a function.